pkg/router/controller/contention.go

```diff
@@ -243,7 +246,25 @@ func (t *SimpleContentionTracker) Clear(id string, current *routev1.RouteIngress
 }
 
 func ingressEqual(a, b *routev1.RouteIngress) bool {
-	return a.Host == b.Host && a.RouterCanonicalHostname == b.RouterCanonicalHostname && a.WildcardPolicy == b.WildcardPolicy && a.RouterName == b.RouterName
+	return a.Host == b.Host && a.RouterCanonicalHostname == b.RouterCanonicalHostname && a.WildcardPolicy == b.WildcardPolicy && a.RouterName == b.RouterName &&
+		cmp.Equal(findMostRecentIngressCondition(a), findMostRecentIngressCondition(b), cmpopts.IgnoreFields(routev1.RouteIngressCondition{}, "LastTransitionTime"))
```

Comment:
Would checking the `Admitted` condition be sufficient?  It seems simpler and easier to reason about.  
```suggestion
		cmp.Equal(findCondition(a, routev1.RouteAdmitted), findCondition(b, routev1.RouteAdmitted), cmpopts.IgnoreFields(routev1.RouteIngressCondition{}, "LastTransitionTime"))
```

Reply:
Yes, that's definitely the preferable approach (for some reason I didn't realize that a route ingress can only have one _admission_ condition). Fixed, thanks!